### ISSUE TITLE
Reproducibility of main results

### ISSUE THREAD

**Issue by jaylee2000** [Shin-woocheol/SQDF#1]
<img width="1463" height="636" alt="Image" src="https://github.com/user-attachments/assets/e218c6e3-3a8b-49d7-8bb6-aa56c8a9c6cc" />

Hello, I am trying to reproduce your main results (5.1 Finetuning for Text-to-Image Diffusion Models, Aesthetics Score) by running `aesthetic.sh`. I did not make any changes to your code, except for changing **train_gradient_accumulation_steps** from 8 to 32, and **train_batch_size** from 4 to 1, in order to fit it inside 24GB GPUs.

However, the reward stagnates at < 6.5 (Update: I have finished training 2000 epochs, and reward did not raise to 8.22).

**Comment by Shin-woocheol** [comment]
Thank you for your interest in our work. We appreciate your detailed report and the GitHub issue you opened.

To address your main concern directly: the Weights & Biases metrics you shared correspond to the training reward mean, not the evaluation metric reported in the paper. 
Specifically, the logged reward is "train reward" computed on samples obtained by one-step denoising from timestep (t) to (t-1), followed by prediction through the consistency model, and then evaluated with the reward model. 
This quantity is used purely as a training signal and is not intended to match the final scores reported in the paper.

Therefore, if you would like to reproduce the scores reported in the paper, please evaluate the aesthetics score on images generated via full denoising from the checkpoints. (see Appendix D.4 for more information.)

Thank you again for your interest and for your careful investigation.
